From 42096f95e39464edcefcf064534dea3163ea74b2 Mon Sep 17 00:00:00 2001
From: Christian Hesse <mail@eworm.de>
Date: Sun, 28 Dec 2014 19:51:47 +0100
Subject: [PATCH 1/2] alpm: bump required libalpm version to 9.0.0

This bumps required libalpm version number to 9.0.0. Please note that we
have to check libalpm version, not pacman version!

Signed-off-by: Christian Hesse <mail@eworm.de>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 2c05359..bc7df9f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -506,7 +506,7 @@ if test x$enable_aptcc = xyes; then
 fi
 
 if test x$enable_alpm = xyes; then
-	PKG_CHECK_MODULES(ALPM, libalpm >= 4.1.0)
+	PKG_CHECK_MODULES(ALPM, libalpm >= 9.0.0)
 fi
 
 if test x$enable_poldek = xyes; then
-- 
2.2.1

